src/views/catalog/wizard/tabs/overview/components/WizardOverviewGrid.tsx

```diff
@@ -128,6 +131,11 @@ const WizardOverviewGrid: React.FC<WizardOverviewGridProps> = ({ vm, tabsData, u
             title={t('CPU | Memory')}
             isEdit
             testId="wizard-overview-cpu-memory"
+            helperPopover={{
+              header: t('CPU | Memory'),
+              content: t(`CPUs = sockets X threads X cores.
+              This VM has ${sockets} sockets, ${threads} threads, and ${cores} cores. `),
```

Comment:
For translations with variables, you should not use literals. 
Instead: 

```suggest
              content: t(`CPUs = sockets X threads X cores.
              This VM has {{sockets}} sockets, {{threads}} threads, and {{cores}}  cores.`, {
                sockets, 
                threads, 
                cores
              }),
``

Reply:
Thank you Ugo!!